fpdcc/ccfp-asset-dashboard
# update phase_type options; update section fixtures; fix cip export bug

## Overview

This PR fixes issues that came up in our weekly check-in.

- Closes #200 
- Closes #157 
- Fixes CIP export bug
- Removed some functionality from the admin section (I didn't think those pieces should be exposed but let me know if you think otherwise)
- showed the phase_type display name in a template

## Testing Instructions
- visit review app
- make sure the sections match the list in #200 
- make sure the phase types match the list in #157 
- export a cip plan where a project has two or more funding streams. export the cip as csv and make sure each funding stream creates a unique row in the csv.
